assets/queries/terraform/azure/network_interfaces_ip_forwarding_enable/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	ip := input.document[i].resource.azurerm_network_interface[name]
+
+	ip.enable_ip_forwarding == true
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_network_interface[%s].enable_ip_forwarding", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("'azurerm_network_interface[%s].enable_ip_forwarding' is set to false", [name]),
```

Comment:
Here, we could indicate `is set to false or undefined` (since the default value is false)